lib/navigation/consts.hpp

```diff
@@ -0,0 +1,46 @@
+#include <cstdint>
+
+
+namespace hyped::navigation{
+
+  using nav_t = float;
+  static constexpr nav_t kTrackLength       = 100.0; //m
+  static constexpr nav_t kBrakingDistance   = 20.0;  //m
+  static constexpr uint8_t kNumberImus      = 4;
+  static constexpr uint8_t kNumberEncoders  = 4;
+  static constexpr uint8_t kNumberKeyence   = 2;
+  static constexpr nav_t kPi                = 3.14159265359;
+  static constexpr nav_t kWheelCicumference = kPi * 0.1; //m //TODO: check!
+  static constexpr nav_t kStripeDistance    = 10.0; //m
+  
+  //current trajectory struct
+  struct Trajectory {
+  nav_t displacement;
+  nav_t velocity;
+  nav_t acceleration;
+};
+
+//inline trajectory equal to oprator for use in testing
+inline bool operator==(const Trajectory &lhs, const Trajectory &rhs)
```

Comment:
Maybe we should move these closer to the tests? There's a problem here because comparing floats with equality doesn't make much sense. I'm not sure the tests are always going to work anyways when we do it.

Reply:
gtest has `ASSERT_FLOAT_EQ` so if this is just for testing, rewrite this within the test/navigation using the asserts

Reply:
This is used in tests contained in several different subdirectories so on second thoughts, probably neater to leave this where it is?